packages/teams-js/test/public/meeting.spec.ts

```diff
@@ -355,201 +511,332 @@ describe('meeting', () => {
+          expect(returnedLiveStreamState).toEqual({ isStreaming: true });
+        });
+      } else {
+        it(`should not allow meeting.registerLiveStreamChangedHandler calls from ${context} context`, async () => {
+          await framedPlatformMock.initializeWithContext(context);
+
+          // eslint-disable-next-line @typescript-eslint/no-empty-function
+          expect(() => meeting.registerLiveStreamChangedHandler(() => {})).toThrowError(
+            `This call is only allowed in following contexts: ${JSON.stringify(
+              allowedContexts,
+            )}. Current context: "${context}".`,
+          );
+        });
+      }
     });
   });
 
   describe('shareAppContentToStage', () => {
     it('should not allow calls before initialization', () => {
-      return expect(() => meeting.shareAppContentToStage('')).toThrowError('The library has not yet been initialized');
-    });
-
-    it('should successfully share app content to stage', () => {
-      desktopPlatformMock.initializeWithContext('sidePanel');
-
-      const requestUrl = 'validUrl';
-      const promise = meeting.shareAppContentToStage(requestUrl);
-
-      const shareAppContentToStageMessage = desktopPlatformMock.findMessageByFunc('meeting.shareAppContentToStage');
-      expect(shareAppContentToStageMessage).not.toBeNull();
-      const callbackId = shareAppContentToStageMessage.id;
-
-      desktopPlatformMock.respondToMessage({
-        data: {
-          id: callbackId,
-          args: [null, true],
-        },
-      } as DOMMessageEvent);
-
-      expect(promise).resolves.toEqual(true);
-      expect(shareAppContentToStageMessage.args).toContain(requestUrl);
-    });
-
-    it('should return error code 500', () => {
-      desktopPlatformMock.initializeWithContext('sidePanel');
-
-      const requestUrl = 'invalidAppUrl';
-      const promise = meeting.shareAppContentToStage(requestUrl);
-
-      const shareAppContentToStageMessage = desktopPlatformMock.findMessageByFunc('meeting.shareAppContentToStage');
-      expect(shareAppContentToStageMessage).not.toBeNull();
-      const callbackId = shareAppContentToStageMessage.id;
-      desktopPlatformMock.respondToMessage({
-        data: {
-          id: callbackId,
-          args: [{ errorCode: ErrorCode.INTERNAL_ERROR }, null],
-        },
-      } as DOMMessageEvent);
-      expect(promise).rejects.toEqual({ errorCode: ErrorCode.INTERNAL_ERROR });
-      expect(shareAppContentToStageMessage.args).toContain(requestUrl);
-      expect;
+      expect(() => meeting.shareAppContentToStage('')).toThrowError('The library has not yet been initialized');
+    });
+
+    const allowedContexts = [FrameContexts.sidePanel];
+    Object.values(FrameContexts).forEach(context => {
+      if (allowedContexts.some(allowedContext => allowedContext === context)) {
+        it('should successfully send the shareAppContentToStage message.', async () => {
+          framelessPlatformMock.initializeWithContext('sidePanel');
```

Comment:
Can you please change the static context string to context variable in required locations.

Reply:
Thank you so much for catching this. It was on my notes to change, but missed somehow. Will do it